data/test/vtgate/unsupported_cases.txt

```diff
@@ -14,10 +14,6 @@
 "explain select * from user"
 "unsupported construct"
 
-# ',' operator for joins
-"select * from user, user_extra"
-"unsupported: ',' join operator"
-
```

Comment:
Can you add in an unsupported case, even though it is redundant with a join?

How about down near line 95
# group by and ,
"select user.id from user, user_extra group by id"
"unsupported: complex join and group by"